README.md

```diff
@@ -106,18 +106,18 @@ respectively.
 
 If you see the screenshot below, you can tell the worker is running:
 
-![](https://github.com/cryptoade1/litentry-parachain/assets/88367184/cb1cea60-bc5d-4b62-bae7-503583a135ee)
+![](https://github.com/cryptoade1/heima/assets/88367184/cb1cea60-bc5d-4b62-bae7-503583a135ee)
 
-### TEE worker tests 
+### Identity worker tests
 
-Refer to [tee-worker ts-tests](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/identity/ts-tests/README.md)
+Refer to [identity-worker ts-tests](https://github.com/litentry/heima/blob/dev/tee-worker/identity/ts-tests/README.md)
 
-### Teardown
+#### Teardown
 
 In the worker launch terminal, `Ctrl + C` should interrupt and clean everything up automatically.
 
-### Additional Info:
+#### Additional Info:
 
-1. Change the RUST_LOG level: `litentry-parachain/local-setup/py/worker.py`
-2. Check existing ts-tests: `litentry-parachain/tee-worker/ts-tests/package.json`
-3. JSON config parameters: `litentry-parachain/tee-worker/service/src/cli.yml`
\ No newline at end of file
+1. Change the RUST_LOG level: `heima/local-setup/worker-log-level-config.toml`
```

Comment:
I think this should be `local-setup/worker-log-level-config.toml`
Relative to README.md root.